config/labels/analyzers/clang-tidy.json

```diff
@@ -194,6 +198,10 @@
       "sei-cert:pos44-c",
       "severity:MEDIUM"
     ],
+    "bugprone-bitwise-pointer-cast": [
```

Comment:
please consider adding all bugprone chekcers to the default profile if they are not too noisy on the open source projects.

Reply:
There are 30 reports, we can enable this.